bedrock/firefox/urls.py

```diff
@@ -73,7 +73,7 @@
     # This dummy page definition makes it possible to link to /firefox/ (Bug 878068)
     url('^firefox/$', views.fx_home_redirect, name='firefox'),
 
-    url('^firefox/os/$', views.firefox_os_index, name='firefox.os.index'),
+    url('^firefox/os/$', views.fxos, name="fxos"),
```

Comment:
I also just realized we're not updating the old view, but are instead ignoring it. Why is that? I believe we should just be replacing the existing `firefox_os_index` view with the new one right?